src/commands-base/base-command.ts

```diff
@@ -30,28 +44,13 @@ export abstract class BaseCommand extends Command {
     this._printContext = { command, flags, args };
   }
 
-  public static serializeFlags<T>(flags: T): T & typeof this.sharedFlags {
+  public static serializeFlags<T>(flags: T): T & typeof sharedFlags {
     return {
-      ...this.sharedFlags,
+      ...sharedFlags,
       ...flags,
-    } as typeof flags & typeof this.sharedFlags;
+    } as typeof flags & typeof sharedFlags;
   }
 
-  static sharedFlags? = {
-    verbose: Flags.boolean({
-      description: 'Print advanced logs (optional).',
-      default: false,
-      helpGroup: 'global',
-    }),
-
-    'print-command': Flags.boolean({
-      aliases: ['pc'],
-      description: 'Print the command that was executed (optional).',
-      default: false,
-      helpGroup: 'global',
-    }),
-  };
-
```

Comment:
optionally: check if you can leave it as a static member of this class, and just use it AS IF it was `export const` -->
BaseCommand.sharedFlags